### DISCUSSION TITLE
import custom properties

### DISCUSSION THREAD

**Discussion by BSalem** [Autodesk/maya-usd#1858] [Q&A] [answered]
how can we import custom properties using -apiSchema flag?

if we have animated userProperties:astPos, then how -apiSchema can import it and create the animation curves?

**Comment by kxl-adsk** [comment, marked as answer]
Answered in the PR - https://github.com/Autodesk/maya-usd/pull/1491#issuecomment-978115045. Feel free to continue the conversation here or in the PR.
